### ISSUE TITLE
ipywidgets 6.0 does not contain PlaceProxy

### ISSUE THREAD

**Issue by nmih** [arose/nglview#590]
Hi all, the newest release of ipywidgets has removed the PlaceProxy widget, see here: https://github.com/ipython/ipywidgets/blob/master/docs/source/changelog.md. Rolling back to version 5.2.2 works fine still.

### Error:
```
      8 import numpy as np
      9 from IPython.display import display
---> 10 from ipywidgets import DOMWidget, widget_image, PlaceProxy
     11 from traitlets import (Unicode, Bool, Dict, List, Int, observe,
     12                        CaselessStrEnum)

ImportError: cannot import name PlaceProxy
```

### Version report:

```bash
python -c 'import nglview; print(nglview.__version__)'
0.6.2.2
python -c 'import ipywidgets; print(ipywidgets.__version__)'
6.0.0
python -c 'import traitlets; print(traitlets.__version__)'
4.2.2
```

**Comment by hainm** [comment]
hi, we fixed it but has not released the new version. Thanks for reporting.

https://github.com/arose/nglview/issues/584

**Comment by hainm** [comment]
We just made a new release, should be available on pypi (all versions) and bioconda (2.7 to 3.5).
cheers.

**Comment by nmih** [comment]
Nice, thank you!
